rust-toolchain.toml

```
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.62.1"
+channel = "1.63.0"
```

```suggestion
channel = "1.63"
```

Best I can tell, the rustup issue only applied to `1.62` :shrug: It's weird. `1.63` seemed to work fine.